src/Misc/layoutbin/en-US/strings.json

```diff
@@ -320,6 +319,7 @@
   "RMDownloadProgressDetails": "{0} MB downloaded at {1} KB/sec. Download time: {2}. Parallel download limit: {3}.",
   "RMDownloadTaskCompletedStatus": "No download tasks have completed in {0} minutes. Remaining task statuses:",
   "RMDownloadTaskStates": " {0}: \t{1} task(s).",
+  "RMEnsureArtifactFolderExist": "Ensuring artifact folder {0} exists.",
```

Comment:
RMEnsureArtifactFolderExistsAndIsClean? Ensuring artifact folder {0} exists and is clean?